go-controller/pkg/ovn/loadbalancer/loadbalancer.go

```diff
@@ -29,6 +49,41 @@ func GetOVNKubeLoadBalancer(protocol kapi.Protocol) (string, error) {
 	return out, nil
 }
 
+// CreateLoadBalancer creates the loadbalancer if it doesn´t exist to avoid
+// consumers to create duplicate loadbalancers with the same name and externalID
+func CreateLoadBalancer(protocol kapi.Protocol, idkey string) (string, error) {
```

Comment:
maybe you've already done this in this PR and I haven't gotten there, but just a note we could replace the creates everywhere witht his new function right?

https://github.com/ovn-org/ovn-kubernetes/blob/master/go-controller/pkg/ovn/master.go#L442